main/webapp/modules/core/langs/translation-en.json

```
@@ -578,6 +580,7 @@
     "core-views/add-id-column": "Add entity identifiers column…",
     "core-views/add-id-column2": "Add a column containing entity identifiers from items",
     "core-views/add-id-col": "Add column containing entity identifiers on \"$1\"",
+    "core-views/add-entity-URL-col":"Add a name for the column containing URL of matched entities",
```

Maybe this text could be simply the same as the action name in the menu entry, "Add column with URLs of matched entities"?
```suggestion
    "core-views/add-entity-URL-col":"Add column with URLs of matched entities",
```